apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.test.js

```diff
@@ -277,4 +282,49 @@ describe('RegisterMultiPaymentForInvoicesService', () => {
             ]))
         })
     })
+
+    describe('RegisterMultiPaymentForInvoicesService minimum amount check', () => {
+
+        let utils
+
+        beforeAll(async () => {
+            utils = new TestUtils([ResidentTestMixin])
+            await utils.init()
+            await utils.updateAcquiringContext({
+                invoiceStatus: CONTEXT_FINISHED_STATUS,
+                invoiceRecipient: createTestRecipient(),
+                invoiceImplicitFeeDistributionSchema: [{
+                    recipient: 'organization',
+                    percent: '5',
+                }],
+            })
+        })
+
+        afterEach(async () => {
+            await utils.updateAcquiringIntegration({ minimumPaymentAmount: null })
+        })
+
+        describe('Check minimum payment amount from acquiring integration', () => {
+
+            test('No limits for payment if no settings for minimumPaymentAmount in acquiring integration', async () => {
+                const [invoice] = await createTestInvoice(utils.clients.admin, utils.organization, { status: INVOICE_STATUS_PUBLISHED })
+                const [result] = await registerMultiPaymentForInvoicesByTestClient(utils.clients.admin, { invoices: [{ id: invoice.id }] })
+                expect(result).toHaveProperty('multiPaymentId')
+            })
+
+            test('The payment amount is less than the minimum amount from acquiring integration', async () => {
```

Comment:
You also should add at least one positive test and one test where minimumPaymentAmount equals amountToPay